packages/meteor-gazelle-createdby/lib/tests/createdByTester.js

```diff
@@ -0,0 +1,12 @@
+CreatedByTesters = new Mongo.Collection('createdByTesters');
+CreatedByTester = Astro.Class({
+  name: 'CreatedByTester',
+  collection: 'CreatedByTesters',
+  fields: {
+    postContent: 'string'
+  },
+  // Validators not should be added by the behavior.
```

Comment:
In fact, I put that comment in because my understanding is that the Validators we need will be included in the `createdby` behavior. (That is, for the createdby and updatedby fields.)